src/dotnet/commands/dotnet-vstest/VSTestForwardingApp.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.Cli
+{
+    public class VSTestForwardingApp
+    {
+        private const string vstestExename = "vstest.console.dll";
+        private readonly ForwardingApp _forwardingApp;
+
+        public VSTestForwardingApp(IEnumerable<string> argsToForward)
+        {
+            _forwardingApp = new ForwardingApp(
+                GetVSTestExePath(),
+                argsToForward,
+                environmentVariables: GetEnvironmentVariables()
+                );
+        }
+
+        public int Execute()
+        {
+            return _forwardingApp.Execute();
+        }
+
+        private Dictionary<string, string> GetEnvironmentVariables()
+        {
+            return new Dictionary<string, string>
+            {
+                { "DotnetHostPath", GetHostPath() },
```

Comment:
My initial impression was that this environment variable is getting use by ForwardingApp class to find dotnet.exe so I added this. I saw that MSBuildForwardingApp class is passing it to ForwardingApp class. I removed this and still every thing is working fine. Its seems my understanding was wrong. 
I will remove it in next commit.